dist/index.js

```diff
@@ -12387,41 +12387,63 @@ const zipQuest = async () => {
   return zipFile;
 }
 
-const uploadQuest = async (questId, zipFile) => {
-  console.log(`Uploading ${questId}`);
+const uploadQuest = async (questId, zipFile, onlyValidate = false) => {
```

Comment:
why dist is here?

Reply:
this done automatically by github

Reply:
Can you just set dist in .ignore?

Reply:
Don't think it's gonna help, It is github's commit